CODEOWNERS

```diff
@@ -0,0 +1,123 @@
+# report on Jira.
+
+# This file will notably establish who is responsible for a specific
+# area of ReactOS. Being a maintainer means the following:
+#	- that person has good knownledge in the area
+#	- that person is able to enforce consistency in the area
+#	- that person may be available for giving help in the area
+#	- that person has push access on the repository
+# Being a maintainer does not mean the following:
+#	- that person is dedicated to the area
+#	- that person is working full-time on the area/on ReactOS
+#	- that person is paid
+#	- that person is always available
+
+# We have no supported (paid) areas in ReactOS.
+
+# When submitting a pull request on GitHub and looking for reviewers,
+# look at that file and ask for a review from some of the people (M, R
+# - the most recently active in the area) listed in the matching area,
+# also, assign the pull request to the M person. Don't ask for a review
+# from all the listed reviewers.
+# Also, when submitted a pull request on GitHub, rules defined in
+# CONTRIBUTING.md apply. And if the maintainer is not available and
+# reviewers approved the pull request, developers feeling confident
+# can merge the pull request. Note that reviewers do not necessarily
+# have push access to the repository.
+# When submitting a bug report on Jira, if you want to be sure to have
+# a developer with skills in that area, write @nick from M people.
+
+# There should be one and only one primary maintainer per area.
+
+# In case of 3rd party code (also refered as upstream), the maintainer
+# is responsible of updating periodically the source code and of
+# managing local patches. He is not here to upstream code on your behalf.
+# As responsible, he may refuse a local patch if you did not try to
+# upstream your changes.
+
+# If you want to get listed in that file, either put yourself in the
+# file and push it, or open a pull request. You can also ask a person
+# who has push access to add yourself.
+
+# This file uses GitHub's format for specifying code owners.
+#	- Lines starting with # are comment lines.
+#	- All other lines specify a path / file (wildcards allowed) followed
+#	  by the GitHub user name(s) of the code owners.
+# See https://help.github.com/en/articles/about-code-owners
+
+# Additionally it tries to use a similar format to the Linux kernel
+# MAINTAINERS file within comments.
+# Descriptions of comment section entries used here:
+#	M: Primary maintainer. Assign them pull requests
+#	   Use the GitHub, Jira, Real Name format for entry, squash if
+#	   some are overlapping
+#	R: Reviewers. Ask them for review on pull requests
+#	S: Status, one of the following:
+#	   Maintained:  Someone is handling that area
+#	   Upstream: This is 3rd party code, synced in our tree
+#	   Abandoned:  No one is handling that code anymore
+#	F: Files. Directories, files (wildcards allowed) covered in
+#	   this area
+#	C: Comments
+
+
+# 3rd Party File Format Libraries
```

Comment:
I would suggest to make a distinction between these functional comments, and the block above explaining stuff.
For example, use double `##` comment markers for the comment block.
This greatly simplifies parsing this file to create a script like `get_maintainer.py`

Reply:
Or prefix the lines that should be read by the script with `#>` or something like that.

Reply:
We can also install the git extension, I linked in the description. I'm not too familiar with how that works and whether we can do that once for everyone, though.